pkg/util.go

```diff
@@ -102,12 +102,14 @@ func SimulateHTTP(ctx context.Context, tc *models.TestCase, testSet string, logg
 	if err != nil {
 		utils.LogError(logger, err, "failed to parse the template")
 	}
-	var output bytes.Buffer
-	err = tmpl.Execute(&output, utils.TemplatizedValues)
-	if err != nil {
-		utils.LogError(logger, err, "failed to execute the template")
+	if tmpl != nil {
```

Comment:
if tmpl is nil, can we do anything in the further flow or should we return an error in this case also that couldn't get the tmp?